flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -72,6 +72,21 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                     .withDescription(
                             "Stabilization period in which no new scaling will be executed");
 
+    public static final ConfigOption<List<String>> EXCLUDED_PERIODS =
+            autoScalerConfig("excluded.periods")
+                    .stringType()
+                    .asList()
+                    .defaultValues()
```

Comment:
```suggestion
                    .noDefaultValue()
```

Reply:
Fixed, can you approve the workflow again, thanks.

Reply:
@1996fanrui change the defaultValue to `noDefaultValue` causes NPE， we have to add many annoying
 notNull judgments, as config `vertex.exclude.ids` also set to defaultValues(), I referred to it, what do you think.

Reply:
It's fine for me.